stdlib/public/runtime/AutoDiffSupport.cpp

```diff
@@ -19,6 +19,7 @@ using namespace llvm;
 
 SWIFT_CC(swift)
 static void destroyLinearMapContext(SWIFT_CONTEXT HeapObject *obj) {
+  static_cast<AutoDiffLinearMapContext *>(obj)->~AutoDiffLinearMapContext();
```

Comment:
Is this related?

Reply:
This is on top of #34900. The diff should disappear once it's merged.